app/src/main/res/drawable/pill_campus_active.xml

```diff
@@ -1,12 +1,12 @@
+        android:pathData="M54.568 27.479c-5.482 0 -8.9 -3.952 -8.9 -10.336 0 -6.399 3.404 -10.35 8.887 -10.35 4.347 0 7.71 2.816 8.203 6.836h-3.049c-0.574 -2.42 -2.557 -3.979 -5.154 -3.979 -3.541 0 -5.756 2.885 -5.756 7.493 0 4.593 2.215 7.478 5.77 7.478 2.61 0 4.593 -1.408 5.14 -3.637h3.049c-0.547 3.897 -3.787 6.495 -8.19 6.495zm15.735 -2.708c2.215 0 3.855 -1.408 3.855 -3.253v-1.245l-3.637 0.233c-2.05 0.137 -2.98 0.834 -2.98 2.146 0 1.34 1.162 2.12 2.762 2.12zm-0.752 2.475c-2.871 0 -4.963 -1.736 -4.963 -4.457 0 -2.68 1.996 -4.225 5.537 -4.443l4.033 -0.233v-1.285c0 -1.572 -1.039 -2.46 -3.049 -2.46 -1.64 0 -2.775 0.6 -3.103 1.653h-2.844c0.3 -2.556 2.707 -4.197 6.084 -4.197 3.733 0 5.838 1.86 5.838 5.004V27h-2.83v-2.092h-0.233c-0.888 1.49 -2.515 2.338 -4.47 2.338zM80.268 27V12.111h2.83v2.393h0.218c0.657 -1.682 2.16 -2.68 4.143 -2.68 2.037 0 3.445 1.053 4.102 2.68h0.232c0.752 -1.627 2.46 -2.68 4.525 -2.68 3.008 0 4.758 1.819 4.758 4.922V27h-2.94v-9.488c0 -2.01 -0.956 -2.994 -2.884 -2.994 -1.9 0 -3.145 1.367 -3.145 3.117V27h-2.884v-9.775c0 -1.682 -1.121 -2.707 -2.871 -2.707 -1.791 0 -3.145 1.462 -3.145 3.377V27h-2.94zm31.662 -15.135c3.787 0 6.18 2.953 6.18 7.684 0 4.73 -2.393 7.697 -6.153 7.697 -2.12 0 -3.8 -0.93 -4.58 -2.502h-0.232v7.205h-2.94V12.111h2.83v2.461h0.219c0.902 -1.695 2.652 -2.707 4.676 -2.707zm-0.834 12.742c2.502 0 3.992 -1.9 3.992 -5.058 0 -3.145 -1.49 -5.045 -3.979 -5.045 -2.474 0 -4.02 1.941 -4.02 5.059 0 3.103 1.546 5.044 4.007 5.044zm22.091 -12.496V27h-2.83v-2.365h-0.232c-0.725 1.695 -2.242 2.652 -4.498 2.652 -3.295 0 -5.127 -2.01 -5.127 -5.51v-9.666h2.94v8.9c0 2.407 0.97 3.555 3.13 3.555 2.38 0 3.678 -1.408 3.678 -3.746v-8.709h2.94zm2.924 4.211c0 -2.638 2.42 -4.498 5.784 -4.498 3.35 0 5.537 1.559 5.824 4.184h-2.817c-0.273 -1.094 -1.34 -1.791 -3.007 -1.791 -1.641 0 -2.885 0.78 -2.885 1.941 0 0.889 0.752 1.45 2.365 1.819l2.475 0.574c2.83 0.656 4.156 1.873 4.156 4.06 0 2.803 -2.611 4.676 -6.166 4.676 -3.541 0 -5.852 -1.6 -6.111 -4.238h2.939c0.37 1.148 1.463 1.86 3.24 1.86 1.832 0 3.131 -0.821 3.131 -2.01 0 -0.89 -0.697 -1.463 -2.187 -1.819l-2.598 -0.601c-2.83 -0.67 -4.143 -1.942 -4.143 -4.157z"
```

Comment:
I'm assuming this change takes care of replacing the icons of the pill? If it's just readjusting the icons, plz ignore

Reply:
Yup! Edited the description to include that